api-admin/src/main/java/bio/terra/pearl/api/admin/service/notifications/ScheduledSendgridActivityService.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.pearl.api.admin.service.notifications;
+
+import java.util.concurrent.TimeUnit;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
```

Comment:
this class could use a top-level comment about what it's about.  Also, it would be nice if the classname made clear this is just getting the activity logs -- it's not an activity source itself.  Maybe `ScheduledSendgridEventFetcher` or somesuch